BUILD.bazel

```diff
@@ -1,4 +1,6 @@
+load("@buildifier_prebuilt//:rules.bzl", "buildifier")
 load("@rules_pkg//:pkg.bzl", "pkg_tar")
+load("//bazel:compat.bzl", "SKIP_ON_WINDOWS")
```

Comment:
We could build `addlicense` for Windows and add a repository for the `clang-format` binary, but `buildifier-prebuilt` currently doesn't support Windows since it runs a shell wrapper around buildifier.

Since our format script is a shell script anyway, I think that supporting Windows would require quite a bit of effort. If we get contributors working on Windows, we may want to point them to WSL instead.